build.gradle

```diff
@@ -8,6 +8,8 @@ architectury {
     minecraft = rootProject.minecraft_version
 }
 
+def usingHotswapAgent = true
```

Comment:
What do we think of defaulting this to false, and having this in as an easy "opt-in"?

Reply:
Yeah, that is perhaps a good solution for now. Or even setting your own default value in `~/.gradle/gradle.properties`.

Reply:
I just wanted to run `java -XX:HotswapAgent=fatjar --version` and check the exit code, but I failed to figure out how to do that. Sometimes I really hate Gradle.